community/kernel/src/test/java/org/neo4j/kernel/impl/util/dbstructure/DbStructureLargeOptionalMatchStructure.java

```diff
@@ -0,0 +1,111 @@
+/*
```

Comment:
This is part of the test. This file represents the relevant subset of the real statistics of a real database that was originally used to reproduce the problem. The fact that the generated statistics structure is saved to kernel, while the test is in cypher is a bit odd, but that is the pattern used by all previous similar tests, so we kept it for consistency.